rllib/evaluation/sampler.py

```diff
@@ -968,9 +944,6 @@ def _process_observations(
                     build=episode.is_faulty or not multiple_episodes_in_batch,
                 )
             if not episode.is_faulty:
-                if ma_sample_batch:
```

Comment:
Without going into depth here: This looks where only returning non-fauly episodes before and now return all episodes. Is that intended?

Reply:
See my comment below on the line of code where this logic was moved to.